.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -0,0 +1,12 @@
+# https://github.com/gradle/wrapper-validation-action#readme
+
+name: "Validate Gradle Wrapper"
```

Comment:
Good question, were we going to accept an external PR with a JAR change?
But then did you check the source code of the action to make sure it was not doing anything suspicious it self 😛

Reply:
I hope so ! Otherwise we should really contact the Spring Boot team https://github.com/spring-projects/spring-boot/blob/master/.github/workflows/gradle-wrapper-validation.yml !